google/cloud/aiplatform/training_jobs.py

```diff
@@ -2859,6 +2898,13 @@ def _run(
 
         training_task_definition = schema.training_job.definition.automl_tabular
 
+        # convert column specs to column transformations
+        if self._column_specs is not None:
```

Comment:
To simplify, I think we can avoid the _column_specs instance attribute and capture this in the constructor above like so. Also it's more readable to name the tuple values in the list comprehension:

 ```Python
if column_transformations is not None:
    self._column_transformations = column_transformations
elif column_specs is not None:
    self._column_transformations = [
        {transformation: {"column_name": column_name}}
        for column_name, transformation in column_specs.items()
    ]
else:
    self._column_transformations = None
```